### PR TITLE
fix wrap error at the pole for siblings and kring quadkey, convert in pure sql, add QUADINT_CENTER

### PR DESCRIPTION
**Pull request by francois-baptiste**
* fix wrap error at the pole for siblings
  * QUADINT_KRING
  * QUADINT_KRING_DISTANCES
  * QUADINT_SIBLING
* convert in pure sql
  * QUADINT_KRING
  * QUADINT_KRING_DISTANCES
  * QUADINT_SIBLING
* add QUADINT_CENTER
* add QUADINT_RESOLUTION

### CODE REVIEW

**Comment by francois-baptiste** [comment]
fixes #204

File: modules/quadkey/bigquery/test/integration/QUADINT_CENTER.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    expect(JSON.stringify(rows[6].boundary.value)).toEqual('"POINT(90 -45)"');
+    expect(JSON.stringify(rows[7].boundary.value)).toEqual('"POINT(-135 40.9798980696201)"');
+    expect(JSON.stringify(rows[8].boundary.value)).toEqual('"POINT(-135 -40.9798980696201)"');
+    expect(JSON.stringify(rows[9].boundary.value)).toEqual('"POINT(-135 -79.1713346408195)"');
+    expect(JSON.stringify(rows[10].boundary.value)).toEqual('"POINT(-44.82421875 44.964797930331)"');
+    expect(JSON.stringify(rows[11].boundary.value)).toEqual('"POINT(-44.9993133544922 45.0002525507932)"');
+    expect(JSON.stringify(rows[12].boundary.value)).toEqual('"POINT(-44.999994635582 44.9999984058533)"');
+});
+
+test('QUADINT_BOUNDARY should fail with NULL argument', async () => {
```
Comment (arredond) [inline modules/quadkey/bigquery/test/integration/QUADINT_CENTER.test.js:30]: Just a typo but you probably want `QUADINT_CENTER` should fail...

**Review by arredond — APPROVED** [review]

File: modules/quadkey/bigquery/test/integration/QUADINT_CENTER.test.js (hunk 1 above)
Comment (francois-baptiste) [reply modules/quadkey/bigquery/test/integration/QUADINT_CENTER.test.js]: Done! Thanks @arredond !

File: modules/quadkey/bigquery/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,17 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [1.1.1] - 2022-07-07
```
Comment (arredond) [inline modules/quadkey/bigquery/CHANGELOG.md:6]: I suppose the month here is wrong, right?

**Review by Jesus89 — COMMENTED** [review]
Good job!

I've found a couple of things:
- QUADINT_KRING/QUADINT_KRING_DISTANCES must allow size=0, so they return the same index with distance 0.
- QUADINT_KRING(0, 1); returns (0), (32), must return only (0)
- QUADINT_KRING_DISTANCES(0, 1); returns  (0, 0), (32, 1), must return only (0, 0)

**Comment by francois-baptiste** [comment]
Thanks for the review @Jesus89 
> * QUADINT_KRING/QUADINT_KRING_DISTANCES must allow size=0, so they return the same index with distance 0.

Ok, for this, I pushed the fix.
> * QUADINT_KRING(0, 1); returns (0), (32), must return only (0)
> * QUADINT_KRING_DISTANCES(0, 1); returns  (0, 0), (32, 1), must return only (0, 0)

I also fixed this (even if quadkey do not support officially level 0, level 1 is the minimum: https://docs.microsoft.com/en-us/bingmaps/articles/bing-maps-tile-system#ground-resolution-and-map-scale)

**Review by Jesus89 — APPROVED** [review]

**Pull request merged**